src/Clock.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './Clock.scss';
+
+export class Clock extends React.Component {
+state = {
+  time: new Date().toLocaleTimeString(),
```

Comment:
you could use .toLocaleTimeString() only while rendering time
```suggestion
  time: new Date(),
```